src/components/nav/NavFooter.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div>
+    <footer class="footer">
+      <div class="container">
+        <div class="column" align="left">
+          <h3>Made at</h3>
+          <a href="https://www.khoury.northeastern.edu/">
+            <img class="khoury-logo" src="../../assets/khoury-logo.png" />
+          </a>
+        </div>
+        <div class="column" align="left">
+          <h3>Upcoming events</h3>
+          <router-link
+            to="../../events"
+            v-for="event in events.slice(0, 3)"
+            :key="event.title"
+          >
+            <p>{{ event.title }}</p>
+          </router-link>
+        </div>
+        <div class="column" align="left">
+          <h3>Find us on</h3>
+          <a
+            href="https://www.facebook.com/c4cneu/"
+            class="fa fa-facebook hvr-grow"
+          ></a>
+          <a
+            href="https://www.instagram.com/c4cneu/"
+            class="fa fa-instagram hvr-grow"
+          ></a>
+          <a
+            href="https://www.linkedin.com/company/code-4-community/about/"
+            class="fa fa-linkedin hvr-grow"
+          ></a>
+          <a href="https://c4cneu.slack.com/" class="fa fa-slack hvr-grow"></a>
+          <a
+            href="https://github.com/Code-4-Community"
+            class="fa fa-github hvr-grow"
+          ></a>
+          <a
+            href="mailto:c4cneu@gmail.com"
+            class="fa fa-envelope-o hvr-grow"
+          ></a>
+        </div>
+      </div>
+    </footer>
+  </div>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      events: this.$store.state.events
+    };
+  }
+};
+</script>
+
+<style scoped>
```

Comment:
wasn't included in the ticket, but can you add styling for mobile? For mobile, let's just have the social links show. Reference the `@media` queries in the App.vue's style tag. Play around with various viewport sizes to see how it looks (I always do this by just resizing my developer tools)

Reply:
nitpick: can we make the icons closer to the "Find us on" text?